src/main/resources/log4j2.xml

```diff
@@ -15,23 +15,62 @@
 ~ See the license for the specific language governing permissions and
 ~ limitations under the license.
 -->
-<Configuration status="ERROR">
+<Configuration status="DEBUG">
     <Properties>
         <Property name="DEFAULT_HOME">$${env:JENKINS_HOME:-$${sys:JENKINS_HOME:-.}}</Property>
         <Property name="FALLBACK_LOG_DESTINATION">${DEFAULT_HOME}/logs/audit.log</Property>
         <Property name="AUDIT_LOG_DESTINATION">$${sys:auditFileName:-${FALLBACK_LOG_DESTINATION}}</Property>
+        <Property name="IANA_ENTERPRISE_NUMBER">$${sys:enterpriseNumber:-.}</Property>
+        <Property name="APPENDER_LAYOUT_TYPE">$${sys:appenderType:-jsonLayout}</Property>
+        <Property name="PORT">1854</Property>
+        <Property name="HOST">localhost</Property>
     </Properties>
+
     <Appenders>
-        <RollingRandomAccessFile name="audit" fileName="${AUDIT_LOG_DESTINATION}" filePattern="${AUDIT_LOG_DESTINATION}.%d{yyyyMMdd_HHmmss}-%i.log">
-            <JsonLayout properties="true"/>
-            <Policies>
-                <SizeBasedTriggeringPolicy size="20 MB"/>
-            </Policies>
-        </RollingRandomAccessFile>
+        <Routing name="Routing">
```

Comment:
Try using https://logging.apache.org/log4j/2.x/manual/appenders.html#ScriptAppenderSelector as a potential alternative? If that works, we could potentially develop a custom log4j plugin similar to it that selects purely based on system properties (or a `pattern` attribute similar to in the routing appender) to avoid evaluating scripts in the config.